packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -126,7 +150,7 @@ export class PendingStateManager implements IDisposable {
 	/**
 	 * Sequenced local ops that are saved when stashing since pending ops may depend on them
 	 */
-	private savedOps: IPendingMessage[] = [];
+	private savedOps: ISavedMessage[] = [];
```

Comment:
This comment is for line 148
> private readonly initialMessages = new Deque < IPendingMessageFromStash >();

Few doubts
1. These messages from Stash will never have a sequence number?
2. Should IMessageToStash generically represent messages from/to stash?